src/TimerOutput.jl

```diff
@@ -61,8 +61,15 @@ function Base.push!(to::TimerOutput, label::String)
     # Fast path
     if to.prev_timer_label == label
         timer = to.prev_timer
+    elseif haskey(current_timer.inner_timers, label)
```

Comment:
Can't we use `get` with a `nothing` default and then we don't have to do the dict lookup twice?

Reply:
Good idea!